### PR TITLE
sim:Add nsh cromfs to build

### PR DESCRIPTION
**Pull request by davids5**
## Summary

Add nsh cromfs to build testing

### CODE REVIEW

File: testlist/sim.dat [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /sim
 -sim:cxxtest
+-sim:nshcromfs
```
Comment (xiaoxiang781216) [inline testlist/sim.dat:3]: this will disable nshcromfs build, does it your intention?

Comment (davids5) [reply testlist/sim.dat:3]: No

**Comment by davids5** [comment]
@liuguo09 @xiaoxiang781216 How abut some documentation? Five line would be helpful

**Comment by xiaoxiang781216** [comment]
Already document here:
https://github.com/apache/incubator-nuttx/blob/master/tools/README.txt#L1043
or run ./tools/testbuild.sh -h

**Comment by davids5** [comment]
Ok Great. I would not have look in a different repo.  I read it, but the usage is still not clear sim.dat does not follow the same pattern as shown in the documentation. 

So are the lists blacklist? (i.e all configs built except for the one listed with -)  or is this the list or all configs and - is a comment

How about adding something like this to the README.md?
```
To add a board to be tested do the following:
1) .....
```

**Comment by xiaoxiang781216** [comment]
> Ok Great. I would not have look in a different repo. I read it, but the usage is still not clear sim.dat does not follow the same pattern as shown in the documentation.
> 
> So are the lists blacklist? (i.e all configs built except for the one listed with -) or is this the list or all configs and - is a comment
> 

No, basically, the list is the whitelist, but you can use - to exclude some config you don't want. # is the comment.

> How about adding something like this to the README.md?
> 
> ```
> To add a board to be tested do the following:
> 1) .....
> ```

If there is something not clear to you, please improve it directly.

**Pull request closed without merging**